### PR TITLE
Add quickstart to README with link to latest

### PR DESCRIPTION
**Pull request by dkristian**
Inspired by @toddmowen adding a link to the version in the userguide, I think it's also useful to link it from the readme.

### CODE REVIEW

**Comment by toddmowen** [comment]
Agree that README is the first place I'd look for the link.

But why are the instructions different to what's in USERGUIDE?

Maybe we should put the sbt instructions in README, and just link to them from USERGUIDE.

**Comment by dkristian** [comment]
@toddmowen Good point. Now that `scalding` depends on `tools` to write the metadata with the features to the sinks, it's sufficient to just depend on `scalding`. I've changed the USERGUIDE to refer to the README, which has been updated. I've also split out the plugin configuration to a new section in the USERGUIDE that explains how to use it to generate metadata. Ready for review again.

**Comment by dkristian** [comment]
I've also piggy-backed a USERGUIDE correction onto this PR (relating to how aggregation is actually done now).

File: USERGUIDE.markdown [hunk 1]
```diff
@@ -1451,6 +1437,26 @@ With blank features, the usual metadata artifacts are generated even though
...
+
+```
+addSbtPlugin("au.com.cba.omnia" %% "coppersmith-plugin" % "<coppersmith-version>")
+```
+
+The latest `<coppersmith-version>` can be found
+[here](http://commbank.artifactoryonline.com/commbank/api/search/latestVersion?g=au.com.cba.omnia&a=coppersmith-core_2.11).
+
+Once configured, metadata can be generated by running the
+`metadata:export` task from SBT. The location of the generated file at
```
Comment (toddmowen) [inline USERGUIDE.markdown]: Missing words "will be logged" in this sentence.

**Comment by toddmowen** [comment]
:+1: apart from typo

Good call moving the plugin to the Advanced section. It's not something that newbies need to try and contend with.

**Pull request merged**